src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -82,6 +90,42 @@ public ReportParser(Configuration configuration) {
 
         return featureResults;
     }
+    
+    /**
+     * If the JSON file has the same scenarios run multiple times, keep only the
```

Comment:
AFAIR JSON does not have results for multiple times. For that case you would rather have more than one JSON file. Can you explain this case?

Reply:
also looks similar to https://github.com/damianszczepanik/cucumber-reporting/pull/894/files#diff-858efbefb85a2d5a42faf0d6a15da70bR64

Reply:
@damianszczepanik - I know this is an old PR. This case happens when we use `TestNGCucumberRunner` and use TestNG's retry analyzer capability to retry tests to disregard flaky tests.

Assuming that we run 1 test with re-run enabled - 

What this does is, in case of flaky behavior, where a test would fail first and then would pass when re-tried - it creates 2 entries in JSON for the same test - 1 for the failed test and 2nd for the same test, re-tried. So the final report shows that there were 2 tests. 

If the re-tried scenario also fails, it is counted as another failed test. so total 2 failed tests
If the re-tried scenario passed, the report says, 1 failed and 1 passed.

This PR seems to solve that problem.

Reply:
Exactly. Sorry I didn't respond earlier, got moved on to other priorities and never had a chance to come back and answer the questions/fixes.